ubuntu_20.04-stable.dockerfile

```diff
@@ -48,7 +48,7 @@ WORKDIR $HOME/opt
 # build MOAB
 RUN mkdir moab \
     && cd moab \
-    && git clone --branch Version5.1.0 --single-branch https://bitbucket.org/fathomteam/moab moab \
+    && git clone --branch 5.3.0 --single-branch https://bitbucket.org/fathomteam/moab moab \
```

Comment:
```suggestion
    && git clone  --single-branch  --branch 5.3.0 --single-branch --depth 1 https://bitbucket.org/fathomteam/moab.git moab \
```

Reply:
> @ngranda - any thoughts on the suggestions from @shimwell ?

Thanks for the ping! I have tested the build with the newer version of moab and I have accepted the changes